src/pybind/mgr/dashboard/frontend/src/app/core/auth/user-form/user-form.component.html

```diff
@@ -1,249 +1,188 @@
-              <div class="{{ passwordStrengthLevelClass }}"
-                   data-toggle="tooltip"
-                   title="{{ passwordValuation }}">
-              </div>
-            </div>
-            <span class="invalid-feedback"
-                  *ngIf="userForm.showError('password', formDir, 'required')"
-                  i18n>This field is required.</span>
-            <span class="invalid-feedback"
-                  *ngIf="userForm.showError('password', formDir, 'passwordPolicy')">
-              {{ passwordValuation }}
-            </span>
-          </div>
-        </div>
-
-        <!-- Confirm password -->
-        <div class="form-group row"
-             *ngIf="!authStorageService.isSSO()">
-          <label i18n
-                 class="cd-col-form-label"
-                 for="confirmpassword">Confirm password</label>
-          <div class="cd-col-form-input">
-            <div class="input-group">
-              <input class="form-control"
-                     type="password"
-                     placeholder="Confirm password..."
-                     id="confirmpassword"
-                     name="confirmpassword"
-                     autocomplete="new-password"
-                     formControlName="confirmpassword">
-              <button type="button"
-                      class="btn btn-light"
-                      cdPasswordButton="confirmpassword">
-              </button>
-              <span class="invalid-feedback"
-                    *ngIf="userForm.showError('confirmpassword', formDir, 'match')"
-                    i18n>Password confirmation doesn't match the password.</span>
-            </div>
-            <span class="invalid-feedback"
-                  *ngIf="userForm.showError('confirmpassword', formDir, 'required')"
-                  i18n>This field is required.</span>
-          </div>
-        </div>
-
-        <!-- Password expiration date -->
-        <div class="form-group row"
-             *ngIf="!authStorageService.isSSO()">
-          <label class="cd-col-form-label"
-                 [ngClass]="{'required': pwdExpirationSettings.pwdExpirationSpan > 0}"
-                 for="pwdExpirationDate">
-            <ng-container i18n>Password expiration date</ng-container>
-            <cd-helper class="text-pre-wrap"
-                       *ngIf="pwdExpirationSettings.pwdExpirationSpan == 0">
-              <p>
-                The Dashboard setting defining the expiration interval of
-                passwords is currently set to <strong>0</strong>. This means
-                if a date is set, the user password will only expire once.
-              </p>
-              <p>
-                Consider configuring the Dashboard setting
-                <a routerLink="/mgr-modules/edit/dashboard"
-                   class="alert-link">USER_PWD_EXPIRATION_SPAN</a>
-                in order to let passwords expire periodically.
-              </p>
-            </cd-helper>
```

Comment:
this helper text is removed from the password expiration unintentionally. Probably this should be added back as it provides some info.